app/utils/wikiFetch.tsx

```diff
@@ -14,15 +14,56 @@ export interface IWikipediaContestantData {
     status: string
 }
 
+interface Section {
+    toclevel: number
+    level: string
+    line: string
+    number: string
+    index: string
+    fromtitle: string
+    byteoffset: number
+    anchor: string
+    linkAnchor: string
+}
+
+interface ParseResult {
+    title: string
+    pageid: number
+    sections: Section[]
+    showtoc: boolean
+}
+
 async function fetchWikipediaData(wikiUrl: string): Promise<IWikipediaData> {
     const response = await fetch(wikiUrl)
     const data = await response.json()
     return data
 }
 
+export async function fetchWikipediaSections(wikiUrl: string): Promise<ParseResult> {
+    const response = await fetch(wikiUrl)
+    const data = await response.json()
+    return data.parse
```

Comment:
Curious, should we do something similar in `fetchWikipedidiaData`? Doesn't need to happen in this PR, just wondering on thoughts 🤔

Reply:
Definitely, if we plan to keep using Wikipedia for multiple shows